cmd/bb_replicator/main.go

```diff
@@ -20,9 +21,17 @@ import (
 
 func main() {
 	if len(os.Args) != 2 {
-		log.Fatal("Usage: bb_replicator bb_replicator.jsonnet")
+		log.Fatal("Usage: bb_replicator [--dump_configuration_struct] bb_replicator.jsonnet")
 	}
 	var configuration bb_replicator.ApplicationConfiguration
+	if os.Args[1] == "--dump_configuration_struct" {
+		if confDump, err := util.MarshalExample(&configuration); err != nil {
+			log.Fatal("Failed to generate configuration example structure: %s", err)
+		} else {
+			fmt.Println(confDump)
```

Comment:
What if instead you'd output the conf in the specified file?

Reply:
Doesn't look much safe, as @EdSchouten pointed out, most of the nested structures are empty.
He could pipe it to a file if wanted to create a base file to work on it.